v2/fake/fake.go

```diff
@@ -172,9 +175,16 @@ func (c *FakeClient) PollLastOperation(r *v2.LastOperationRequest) (*v2.LastOper
 	c.Mutex.Lock()
 	defer c.Mutex.Unlock()
 
+	var lastActionType ActionType
```

Comment:
I don't think we should make assumptions about the ordering of operations - why not do this based on the actual operation key?

Reply:
I like the idea!

Reply:
Oh so setback, OperationKey is on the response, not the request. I only have the request.